### PR TITLE
Reformat code

### PR DESCRIPTION
**Pull request by numas13**
Reduce boilerplate code and reformat match for better readability.

### CODE REVIEW

File: src/write/elf/object.rs [hunk 1]
```diff
@@ -155,6 +155,10 @@ impl<'a> Object<'a> {
     }
 
     pub(crate) fn elf_translate_relocation(&mut self, reloc: &mut Relocation) -> Result<()> {
+        use Architecture as A;
```
Comment (philipc) [inline src/write/elf/object.rs:158]: I don't think this makes enough difference.

File: src/write/elf/object.rs [hunk 2]
```diff
@@ -155,6 +155,10 @@ impl<'a> Object<'a> {
...
+        use RelocationEncoding as E;
+        use RelocationKind as R;
```
Comment (philipc) [inline src/write/elf/object.rs:160]:
```suggestion
        use RelocationKind as K;
```

File: src/write/elf/object.rs [hunk 3]
```diff
@@ -166,280 +170,175 @@ impl<'a> Object<'a> {
             return Ok(());
         };
 
+        let unsupported_reloc = || Err(Error(format!("unimplemented relocation {:?}", reloc)));
```
Comment (philipc) [inline src/write/elf/object.rs:173]: Good idea.

File: src/write/elf/object.rs [hunk 4]
```diff
@@ -166,280 +170,175 @@ impl<'a> Object<'a> {
...
+
+        #[rustfmt::skip]
```
Comment (philipc) [inline src/write/elf/object.rs:175]:
```suggestion
```
I don't think manual formatting is necessary here, and it makes it more difficult to change this code in future. I think avoiding line wrapping is enough.

File: src/write/elf/object.rs [hunk 5]
```diff
@@ -166,280 +170,175 @@ impl<'a> Object<'a> {
...
-                    elf::R_AARCH64_PREL16
-                }
-                (RelocationKind::Relative, RelocationEncoding::AArch64Call, 26)
-                | (RelocationKind::PltRelative, RelocationEncoding::AArch64Call, 26) => {
-                    elf::R_AARCH64_CALL26
-                }
-                _ => {
-                    return Err(Error(format!("unimplemented relocation {:?}", reloc)));
-                }
+            A::Aarch64 => match (size, kind, encoding) {
```
Comment (philipc) [inline src/write/elf/object.rs:177]: Once these are changed back to using rustfmt, I think `size` last will still be better.

**Review by philipc — CHANGES_REQUESTED** [review]

**Comment by numas13** [comment]
@philipc done.

**Review by philipc — APPROVED** [review]
Thanks!

**Pull request merged**